build.gradle

```diff
@@ -21,7 +21,7 @@ apply plugin: 'maven-publish'
 apply plugin: 'com.jfrog.bintray'
 
 group 'uk.gov.hmcts.reform'
-version '6.0.0'
+version '7.0.0'
```

Comment:
Replace this with the code that looks up version from tag? See if idam-java-client ?

Reply:
OK. was thinking about it but left aside to not mess things up. let me re-work this pr then